content/docs/guides/tips-n-tricks.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Tips, Tricks and Gotchas"
+weight: 4
+description: |
+  Tips and tricks for small places.
+  How to write efficient embedded code and avoid common mistakes.
+---
+
+## Ensure Concurrency
+
+TinyGo code runs on a single core, in a single thread (think `GOMAXPROCS=1`).
+This means we almost always have less threads than goroutines.
```

Comment:
> This means we almost always have less threads than goroutines.

True (there could in theory be zero goroutines and one thread), but not sure it is helpful? I think removing this line might avoid some confusion.

Reply:
I believe, I was citing your slack message. But yes, less confusing text, the better. I remove.